### PR TITLE
fix/dim_students

### PR DESCRIPTION
**Pull request by allison-code-dot-org**
added races field back in in base and staging. Once dim_users is merged, should be pulled in on select *.

### CODE REVIEW

File: dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql [hunk 1]
```diff
@@ -4,53 +4,89 @@
...
+            when gender = 'n' then 'nb' ------  non-binary
+            when gender = 'o' then 'nl' ------  not listed
+            ----  empty string or '-'
+            when gender = '' or gender = '-' then 'no_response'
+            when gender is null then 'not_collected'
+            else 'unexpected value: ' || gender
+        end as gender_group,
+        case
+            -- If races contains 'hispanic', return 'hispanic'
+            when races like '%hispanic%' then 'hispanic'
```
Comment (jordan-springer) [inline dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql:69]: is there a list of accepted values for race? if so, we wouldn't need to fuzzy match here. could also use the accepted values test to handle the `else` condition

File: dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql [hunk 2]
```diff
@@ -4,53 +4,89 @@
...
+
         last_sign_in_at,
+
+        -- misc.
         created_at,
-        updated_at,  
-        deleted_at,   
-        purged_at
+        updated_at,
+        deleted_at,
```
Comment (jordan-springer) [inline dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql:49]: why would we want to include purged and/or deleted users in this model?

**Review by jordan-springer — CHANGES_REQUESTED** [review]
few things

File: dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql (hunk 1 above)
Comment (bakerfranke) [reply dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql:35]: there is a set.  It's 7 or eight possible items, and/but they can be permuted as a comma-separated list in 7! ways.  So fuzzy match is necessary.

File: dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql (hunk 2 above)
Comment (bakerfranke) [reply dbt/models/staging/dashboard_pii/stg_dashboard_pii__users.sql]: There's some difference between delete and purge.  Of fields in users that we're bringing in there's plausible analytic need for these fields, but I don't think its something we'd be moving in to a user-facing model immediately.

**Review by bakerfranke — COMMENTED** [review]
Answered some questions from jordan.  Did not thoroughly review after my contribution, but I think it LGTM.

**Review by bakerfranke — APPROVED** [review]

**Comment by jordan-springer** [comment]
@allison-code-dot-org can we make a decision on which fields to pull from `users_pii` vs `users`? the two models are almost identical atm...

**Pull request merged**